src/navx_node.cpp

```diff
@@ -36,10 +42,12 @@ double frequency;
 bool publish_euler;
 std::string device_path;
 std::string frame_id;
+int covar_samples;
 
 // Global_Vars
 AHRS* com;
 int seq = 0;
+std::vector<Pose> poseHistory;
```

Comment:
There needs to be a prototype definition of calculate_covariance up here. For super-long functions (I mean, holding y'all to a lower quality standard, this shouldn't happen too often naming-wise), this is a good form:
```
void calculate_covariance(
  boost::array<double, 9> &orientation_mat,
  boost::array<double, 9> &ang_vel_mat,
  boost::array<double, 9> &accel_mat
) {
```